### PR TITLE
add tpv36_37 benchmark

### PR DESCRIPTION
**Pull request by fabian-kutschera**

### CODE REVIEW

File: tpv36_37/parameters.par [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+/
+
+&SourceType
+/
+
+&SpongeLayer
+/
+            
+&MeshNml
+MeshFile = 'tpv36_37_mesh_refined.puml.h5'
```
Comment (davschneller) [inline tpv36_37/parameters.par:60]: A `generating_the_mesh.sh` script would be great to have here.

File: tpv36_37/parameters.par [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+&Pickpoint
+printtimeinterval = 1       ! Index of printed info at timesteps
+OutputMask = 1 1 1 1 1 1 1 1 1 1 1 1        ! turn on and off fault outputs
+PPFileName = 'onfault_receivers.dat'
+/
+
+&SourceType
+/
+
+&SpongeLayer
```
Comment (davschneller) [inline tpv36_37/parameters.par:56]: These sections should in principle be obsolete by now

File: tpv36_37/fault_tpv36.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      else 
+        forced_rupture_time = 1000000000.0
+      end
+
+      pi = 4.0 * math.atan(1.0)
+      depth = math.abs(x["z"])
+      w = depth/math.sin(15.0*pi/180.)
+
+      if (w < 8000) then
+        cohesion = -500.0 * math.max(8000.0-w,0)
```
Comment (davschneller) [inline tpv36_37/fault_tpv36.yaml:28]: Is that the only difference here compared to tpv37?

Maybe there's a way to make it an input constant somehow? And then include most of the code presented here? (to reduce code duplication with the `fault_tpv37.yaml` script; though it's not a big issue IMO)

**Review by davschneller — COMMENTED** [review]
Hi, thanks alot for the PR. I've left some tiny comments out there—mainly the existence of a mesh generation script would be great.

File: tpv36_37/parameters.par (hunk 2 above)
Comment (fabian-kutschera) [reply tpv36_37/parameters.par:56]: Thanks, deleted.

File: tpv36_37/fault_tpv36.yaml (hunk 3 above)
Comment (fabian-kutschera) [reply tpv36_37/fault_tpv36.yaml:28]: Yes, the cohesion near the surface is the only difference.
I added a note to the README.md file.

File: tpv36_37/fault_tpv37.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+      r_crit = 4000.0
+      r = math.sqrt((x["x"]-xh)^2 + (x["y"]-yh)^2 + (x["z"]-zh)^2 )
+      Vs = 3464.0
+      if (r <= r_crit) then
+        forced_rupture_time = r/(0.7*Vs)+(0.081*r_crit/(0.7*Vs))*(1.0/(1.0-(r/r_crit)^2)-1.0)
+      else 
+        forced_rupture_time = 1000000000.0
+      end
+
+      pi = 4.0 * math.atan(1.0)
```
Comment (davschneller) [inline tpv36_37/fault_tpv37.yaml:23]: `math.pi` ?

**Review by davschneller — APPROVED** [review]
LGTM.

I've left one more comment regarding `math.pi`; but that can also be left out for now; if that would distort any numerical results or need additional verification.

Also, one more general comment; though that applies to the whole repository: `-` is nicer to type than `_`. :)
But that doesn't need to be changed now. (changing that is probably something that'll need a bit of a broader discussion)

**Pull request merged**